Src/Notion.Client/NotionClient.cs

```diff
@@ -6,6 +6,7 @@ public interface INotionClient
         IDatabasesClient Databases { get; }
         IPagesClient Pages { get; }
         ISearchClient Search { get; }
+        IBlocksClient Blocks { get; }
```

Comment:
@Kurupapuru if you could create a separate PR for this fix would be great so I can merge that and create a release.

Reply:
Thank you for catching this.

Reply:
Saw that this has already been fixed, again sorry for late response
#35